app/addListing/service.js

```diff
@@ -0,0 +1,61 @@
+function service($http, endPoint) {
+    return {
+        addTreatmentCenter: function (scope, auth, formdata, file) {
+            var data = {'user': scope.addtreatmentcenteruser, 'treatment_center': scope.addtreatmentcenter};
+            var req = $http({
+                url: endPoint + '/registrations',
+                //dataType: "json",
+                method: "POST",
+                data: data,
+                //data:formdata,
+                headers: {'Content-Type': 'application/json', 'Authorization': auth}
+            });
+
+            return req.then(function (res) {
+                var statusCode = res.status;
+                //console.log('fdsf'+res);
+                return res.data;
+            });
+        },
+        getAuthtoken: function () {
```

Comment:
Can we have a user service to do this? for signin and signup, all user related services?